gtk/transmission-gtk.appdata.xml.in

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2014 Richard Hughes <richard@hughsie.com>
+Copyright 2017 Endless Mobile, Inc.
```

Comment:
I'm fine with including the script, but this seems like a separate discussion (see #64). And someone will have to keep it up-to-date...